modules/gitops_configuring_sensitive_annotations_in_the_argo_cd_web_ui.adoc

```diff
@@ -0,0 +1,43 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/masking-sensitive-annotations-in-the-argo-cd-web-ui.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops_configuring_sensitive_annotations_in_the_argo_cd_web_ui_{context}"]
+= Configuring sensitive annotations in the Argo CD Web UI
+
+Configure sensitive annotations in the Argo CD Web UI by performing the following steps.
+
+.Procedure
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Edit your ArgoCD CR by completing the following steps:
+.. Under the `spec.extraConfig` section, add the `resource.sensitive.mask.annotations` key.
+.. Specify the annotation keys to mask as a comma-separated list.
+
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example
+spec:
+  extraConfig:
+    resource.sensitive.mask.annotations: openshift.io/token-secret.value, api-key, token #<1>
+----
+<1> - Specify comma-separated list of sensitive annotation values such as, `api-key` and `token`.
++
+. Verify that the value in the Argo CD resource is updated in the `argocd-cm` config map successfully by completing the following steps.
+.. In the *Administrator* perspective, go to *Workload* → *ConfigMaps*.
+.. In the *Project* option, select the `Argo CD` namespace.
+.. In the `argocd-cm` config map, you can now view the `resource.sensitive.mask.annotations` field.
```

Comment:
```suggestion
Verify that the value in the Argo CD resource is updated successfully by completing the following steps.
.. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
.. In the *Project* option, select the `Argo CD` namespace.
.. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
.. Verify that *Status* field of the ArgoCD instance shows as *Phase: Available*.
```
Update the verification steps if we don't want to mention about `argocd-cm` in downstream doc.

Reply:
Fixed.